portafly/src/tests/components/shared/data-list/modals/DataListModal.test.tsx

```diff
@@ -1,16 +1,16 @@
 import React from 'react'
-import { DataListModal } from 'components/data-list/modals'
+
 import { render } from 'tests/custom-render'
 import { fireEvent } from '@testing-library/react'
+import { DataListModal } from 'components'
 
 it('shows up to 5 items and a button to expand the list', () => {
   const items = new Array(10).fill('test').map((admin, i) => admin + i)
   const { getByText, getAllByText } = render(<DataListModal
-    actions={[]}
+    submitButton={<input key="submit" />}
```

Comment:
Maybe this change alongside the ones in `ChangeStateModal.test.tsx` should be in a separated commit (if not a different PR)

Reply:
Totally agree, I'll remove it